tests/queries/0_stateless/02013_json_function_null_column.sql

```diff
@@ -6,3 +6,4 @@ SELECT JSONExtract(toNullable('{"string_value":null}'), 'string_value', 'Nullabl
 SELECT JSONExtract(toNullable('{"string_value":null}'), 'string_value', 'String') as x, toTypeName(x);
 SELECT JSONExtract(NULL, 'string_value', 'Nullable(String)') as x, toTypeName(x);
 SELECT JSONExtract(NULL, 'string_value', 'String') as x, toTypeName(x);
+SELECT JSONExtractString('["a", "b", "c", "d", "e"]', idx) FROM (SELECT arrayJoin([2, NULL, 2147483646, 65535, 65535, 3]) AS idx);
```

Comment:
Let's also add tests with other JSONExtract* functions (e.g. JSONExtractUInt, JSONExtractFloat)

Reply:
BTW It can be copied from #29746

Reply:
Done.